src/diffusers/models/transformers/transformer_chronoedit.py

```diff
@@ -568,9 +576,10 @@ class ChronoEditTransformer3DModel(
         "blocks.0": {
             "hidden_states": ContextParallelInput(split_dim=1, expected_dims=3, split_output=False),
         },
-        "blocks.*": {
-            "encoder_hidden_states": ContextParallelInput(split_dim=1, expected_dims=3, split_output=False),
-        },
+        # NOTE(DefTruth): We need to disable the splitting of encoder_hidden_states because 
```

Comment:
```suggestion
        # Reference: https://github.com/huggingface/diffusers/pull/12660
        # We need to disable the splitting of encoder_hidden_states because 
```